src/Sarif.Viewer.VisualStudio/Models/SarifErrorListItem.cs

```diff
@@ -399,62 +399,62 @@ internal void RemapFilePath(string originalPath, string remappedPath)
 
         internal void AttachToDocument(string documentName, long docCookie, IVsWindowFrame pFrame)
         {
-            LineMarker.AttachToDocument(documentName, (long)docCookie, pFrame);
+            LineMarker.AttachToDocument(documentName, docCookie, pFrame);
```

Comment:
You're right. Good catch. I misread that as a static method call. Will fix, then merge.